podcasts/End of Year/Stories/TopCategoriesStory/CategoryPillar.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+struct CategoryPillar: View {
+    let color: Color
+    let text: String
+    let title: String
+    let subtitle: String
+    let height: CGFloat
+
+    var body: some View {
+        VStack {
+            ZStack {
+                ZStack {
+                    VStack {
+                        Text(title)
+                            .multilineTextAlignment(.center)
+                            .font(.system(size: 14, weight: .heavy))
+                            .foregroundColor(.white)
+                            .frame(width: 90)
+                            .fixedSize()
+                        Text(subtitle)
+                            .multilineTextAlignment(.center)
+                            .font(.system(size: 12, weight: .bold))
+                            .foregroundColor(.white)
+                            .opacity(0.8)
+                            .frame(width: 90)
+                            .padding(.bottom)
+                            .fixedSize()
+
+                        ZStack(alignment: .top) {
+                            ZStack {
+                                Image("square_perspective")
+                                    .resizable()
+                                    .renderingMode(.template)
+                                    .frame(width: 90, height: 52)
+                                    .foregroundColor(color)
+
+                                let whiteContrast = color.contrast(with: .white)
+                                let textColor = whiteContrast < 2 ? UIColor.black.color : UIColor.white.color
+
+                                let values: [CGFloat] = [1, 0, 0.50, 1, 0, 0]
+                                VStack {
+                                    Text("\(text) ")
```

Comment:
I added that because "1" wasn't rendering correctly on iPhone 8. I just tried and guess what? Now it works.

Changed in baf40b8df0791a983a3d0c094aad681e07c5967f